packages/es-components/src/components/containers/banner/useBanner.js

```diff
@@ -0,0 +1,59 @@
+/* eslint react/prop-types: 0 */
+import React from 'react';
+import styled from 'styled-components';
+import { useTheme } from '../../util/useTheme';
+
+const ContentWrapper = styled.div`
+  align-self: center;
+  flex-grow: 1;
+  word-break: break-word;
+`;
+
+function BannerContent(props) {
+  const { children, ...rest } = props;
+  return (
+    <>
+      <ContentWrapper {...rest}>{children}</ContentWrapper>
+    </>
+  );
+}
+
+const Banner = styled.div`
+  align-items: center;
+  background-color: ${props => props.variant.bgColor};
+  border-radius: 2px;
+  color: ${props => props.variant.textColor};
+  display: flex;
+  margin-bottom: 25px;
+  padding: 15px;
```

Comment:
This margin/padding difference feels very unbalanced to me, looking at the image in the description. (@wtw-austinandriese can correct me if I'm wrong on that.)

We really shouldn't be using pixels for padding and margin on something like this, it should either be `em` (so that the padding scales with the font size), or should come from the theme.
```suggestion
  padding: ${({theme}) => theme.spacing.defaultMargin}
```